src/Common/src/System/Drawing/ColorTable.cs

```diff
@@ -21,10 +23,15 @@ internal static class ColorTable
 
         private static void FillWithProperties(Dictionary<string, Color> dictionary, Type typeWithColors)
         {
+            object? value;
```

Comment:
I'd suggest reverting these changes and just adding the `!` below, but if you decide to keep the assert, you don't need to declare this here... you can just declare it below where it's initialized, e.g. `object? value = prop.GetValue(null, null);`